src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+
+  &--indent:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
```

Comment:
don't do it in the modifier, it's the same as you set it to the block directly. Use mix instead